react-charts/donut-utilization-chart/step5.md

```diff
@@ -1,32 +1,61 @@
-Now we'll add tooltips to the donut utilization chart...
+Add a legend to the donut utilization chart.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the code for the `<Chart>` component.</strong>
 
-2) Locate the code that looks like the following:
+It should look like this:
 
 <pre class="file">
 &lt;ChartDonutUtilization
+  constrainToVisibleArea
   data={{ x: &#39;Storage capacity&#39;, y: 45 }}
   title=&quot;45%&quot;
   subTitle=&quot;of 100 GBps&quot;
+  labels={({ datum }) =&gt; datum.x ? `${datum.x}: ${datum.y}%` : null}
   thresholds={[{ value: 60 }, { value: 90 }]}
   height={230}
   width={350}
 /&gt;
 </pre>
 
-3) Add the following properties to that section:
+3) <strong>Add the `padding` property to that section:</strong>
+
+In this case, give it the following values. More padding will be added to the right property to accommodate the axis labels.
+
+- bottom: 20
+- left: 20
+- right: 290
+- top: 20
+
+It should look like this:
 
 <pre class="file" data-target="clipboard">
-constrainToVisibleArea
-labels={({ datum }) =&gt; datum.x ? `${datum.x}: ${datum.y}%` : null}
+padding={{
+  bottom: 20,
+  left: 20,
+  right: 290,
+  top: 20
+}}
 </pre>
 
-- The `constrainToVisibleArea` property will alter the position of the tooltip so that it exactly fits within the rendered SVG
+4) <strong>Add the `LegendOrientation` and `legendPosition` properties to the component.</strong>

```

Comment:
```suggestion
4) <strong>Add the `legendOrientation` and `legendPosition` properties to the component.</strong>
```

Reply:
Split up legend orientation into one step
Legend position in another step
and adding legend data in a third step